streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -269,6 +270,33 @@ class StreamingContext private[streaming] (
     RDDOperationScope.withScope(sc, name, allowNesting = false, ignoreParent = false)(body)
   }
 
+  private[streaming] val recoverableAccuNameToAcc: mutable.Map[String, Accumulable[_, _]] =
+    mutable.Map.empty
+
+  /**
+    * Different from accumulator in SparkContext, it will first try to recover from Checkpoint
+    * if it exist.
+    *
+    * @param initialValue   initial value of accumulator. It will be ignored when recovering from
```

Comment:
OK, and there are still many counter cases in the same file, and I will change them all.